pkg/addons/data/ClusterRoleBinding.rbac.authorization.k8s.io/geneva-cluster-reader.yaml

```diff
@@ -13,3 +13,6 @@ subjects:
 - kind: ServiceAccount
   name: geneva
   namespace: openshift-azure-monitoring
+- kind: ServiceAccount
```

Comment:
Need to think about how to limit this. Prom operator needs access to read endpoints, svc, and get to the metrics

Reply:
It appears this cluster role binding binds cluster-reader to this service account.  That has access to read everything minus secrets.

If we could list the dependencies then we could come up with a policy to limit the access.  What does get metrics require?

Start with something small and play whack-a-mole until the pieces come together.  @mjudeikis, let me know how I can help.
```
rules
- apiGroups:
  - ""
  resources:
  - endpoints
  - svc
  verbs:
  - get
  - list
```

Reply:
removed

Reply:
this is granting prometheus-k8s cluster reader.  I do not want to do that.